.biased_lang_exclude

```diff
@@ -1,6 +1,41 @@
+.biased_lang_exclude
 pkg/splunk/common/urls.go
 pkg/splunk/common/urls_testingcode.go
 pkg/splunk/common/paths.go
+PROJECT
+ChangeLog.md
+.rgignore
+# Exceptions for Obsolete CRDs
+config/crd/bases/
+bundle/manifests/
+config/rbac/role.yaml
+#Exceptions for dual support of license manager
```

Comment:
Using specific files for exceptions rather than folders such `pkg/*`
Now with the new CRDs, we're tightening the grip on `biasLang` workflow.